addon-test-support/@ember/test-helpers/setup-application-context.ts

```diff
@@ -87,17 +88,17 @@ export function setupRouterSettlednessTracking() {
   HAS_SETUP_ROUTER.set(context, true);
 
   let { owner } = context;
-  let router;
+  let router: Router;
   if (CAN_USE_ROUTER_EVENTS) {
-    router = owner.lookup('service:router');
+    router = owner.lookup('service:router') as Router;
```

Comment:
Is it a pain (or just not really necessary) to do an assertion here?

Reply:
That's a very good call. I'll update!